app/stac_api/views.py

```diff
@@ -146,11 +150,12 @@ class AssetsList(generics.GenericAPIView):
     queryset = Asset.objects.all()
     pagination_class = None
 
-    # def get_queryset(self):
-    #     # filter based on the url
-    #     return Asset.objects.filter(
-    #         collection=self.kwargs['collection_name'], feature=self.kwargs['item_name']
-    #     )
+    def get_queryset(self):
+        # filter based on the url
+        return Asset.objects.filter(
+            collection__collection_name=self.kwargs['collection_name'],
+            feature__item_name=self.kwargs['item_name']
```

Comment:
filtering on the item_name should be sufficient, otherwise we might need a combined index for perf reasons.

Reply:
True, I'll remove the filter on collection name.